src/utils/viewer/viewer.cpp

```diff
@@ -525,6 +549,106 @@ void Viewer::switchCameraType() {
   }
 }
 
+void Viewer::saveCameraTransformToFile() {
+  if (!Cr::Utility::Directory::exists(saved_transformations_directory)) {
+    Cr::Utility::Directory::mkpath(saved_transformations_directory);
+  }
+
+  // update temporary save
+  lastCameraSave_ = renderCamera_->node().absoluteTransformation();
+  cameraSaved = true;
+
+  // update save in file system
+  saveNodeTransformToFile(renderCamera_->node(),
+                          saved_transformations_directory + "camera." +
+                              getCurrentTimeString() + ".txt");
+}
+
+void Viewer::loadCameraTransformFromFile() {
+  if (!Cr::Utility::Directory::exists(saved_transformations_directory)) {
```

Comment:
Like I said, the user should have the freedom to load any file store anywhere in her file system.

Reply:
Okay, my fix was to simply remove this conditional, because if the file doesn't exist it'll be caught by file.good() later. Let me know if something else would be better.